patches/react-native+0.61.5.patch

```diff
@@ -21,6 +21,20 @@ index 65fa2bb..c5f265b 100644
      index++;
    }
  
+diff --git a/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m b/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
```

Comment:
We'll need to duplicate this patch for WPiOS as well. You can check here to see how it's done for other patches: https://github.com/wordpress-mobile/WordPress-iOS/blob/e221f2345d1e953737458ed1559621df030683af/Podfile#L223

Reply:
True! Thanks for the pointer 👍